pySDC/tests/test_tutorials/test_step_7.py

```diff
@@ -127,3 +127,27 @@ def test_D():
     from pySDC.tutorial.step_7.D_pySDC_with_PyTorch import train_at_collocation_nodes
 
     train_at_collocation_nodes()
+
+
+@pytest.mark.firedrake
+def test_E():
+    from pySDC.tutorial.step_7.E_pySDC_with_Firedrake import runHeatFiredrake
+
+    runHeatFiredrake(useMPIsweeper=False)
+
+
+@pytest.mark.firedrake
```

Comment:
Don't you need to mark this as parallel test here?

Reply:
Firedrake comes with MPI, so the environment necessarily has everything for parallel runs.